src/select/README.md

```diff
@@ -0,0 +1,87 @@
+# select
+Enhances the native HTML `<select>` with styling and extra features.
+
+## Usage
+
+```html
+<!-- simple -->
+<ssv-select 
+    options.bind="options"
+    selected.bind="selectedItem"
+    placeholder="select game">
+</ssv-select>
+
+<!-- basic options -->
+<ssv-select 
+    options.bind="heroes"
+    text="name"
+    value="id"
+    selected.bind="heroSelected"
+    placeholder="select hero">
+</ssv-select>
+
+<!-- multi select -->
+<ssv-select options.bind="options" 
+    type="multi"
+    auto-close="false"
+    placeholder="select game"
+    selected.bind="optionsSelected"
+    color="primary">
+</ssv-select>
+
+<!-- handling select change event -->
+<ssv-select options.bind="options"
+    placeholder="select game"
+    change.delegate="onChange($event)">
+</ssv-select>
+```
+
+```ts
+onChange(event: CustomEvent) {
+    console.log("onChange::triggered", {
+        previous: event.detail.previous,
+        value: event.detail.value
+    });
+}
+```
+
+### Attributes
+
+| Name            | Type             | Default      | Required | Description                                                                                                              |
+|-----------------|------------------|--------------|----------|--------------------------------------------------------------------------------------------------------------------------|
+| options         | `any[]`          | -            | yes      | Options available in the list. This could be a primitive array (eg: `string[]`) or complex type array (eg: `Hero[]`).    |
+| selected        | `any` or `any[]` | -            | yes      | Currently selected options.                                                                                              |
+| type            | `SelectType`     | `single`     | no       | Can be `single` or `multi`. This will allow single or multi selection.                                                   |
+| placeholder     | `string`         | -            | no       | Placeholder text to show when input is empty.                                                                            |
+| text            | `string`         | `"text"`     | no       | Text property name, which will be used as a select display text.                                                         |
```

Comment:
perhaps renamed these to `textField` and `valueField` + align with global configs